packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -201,8 +199,20 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
     link.url,
   ]);
 
+  const onChangeRel = useCallback(
+    (value) => {
+      const rel = link?.rel?.includes(value)
+        ? link?.rel?.filter((el) => el !== value)
+        : [...link?.rel, value];
+      handleChange({ rel }, true);
+    },
+    [handleChange, link]
+  );
```

Comment:
All the optional chaining here is superfluous and not done anywhere else in this file.

If `link` or `link.rel` can be `null`, then `[...link?.rel, value]` (same as `[...null, value]`) would throw an error.

Assuming that `link.rel` can be `null` for existing links in existing stories, then it'd be safer to write something like 

```js
const linkRel = link.rel || [];
const rel = linkRel.includes(value) ? linkRel.filter((el) => el !== value) : [...linkRel, value];
handleChange({ rel }, true);
```

Reply:
Went a different direction with this.